generators/client/templates/angular/package.json.ejs

```diff
@@ -178,14 +178,17 @@ limitations under the License.
     "clean-www": "rimraf <%= DIST_DIR %>app/{src,<%= BUILD_DIR %>}",
     <%_ if (protractorTests && !cypressTests) { _%>
     "e2e": "<%= clientPackageManager %> run e2e:protractor",
+    "e2e:headless": "<%= clientPackageManager %> run e2e:protractor:headless",
```

Comment:
Instead of too many scripts, can we just limit to `e2e` and `e2e:headless` (irrespective of the choice `protractor` or `cypress`)?

Reply:
Currently we can generate both in the same project.

Reply:
It felt easier to maintain it as well that way

Reply:
Yes but we can't, both Protractor and Cypress can be chosen ATM.

Reply:
@avdev4j, do we have a use case that require generation of both in same project?

Reply:
Of course not, at least if you don't want to fall into madness. But it does exist, I mean it possible to choose both so we decide to handle those case adding 2 scripts per each.